index.mako

```diff
@@ -445,7 +445,8 @@ file that tells pre-commit:
 =r=
     =c= [`pass_filenames`](_#hooks-pass_filenames)
     =c= (optional: default `true`) if `false` no arguments will be passed to
-        the hook.
+        the hook. Set this to `True` when creating a local hook and passing arguments through
+        [`entry`](#hooks-entry).
```

Comment:
this isn't quite right, it has nothing to do with that -- it's only about passing filenames along

Reply:
I see, maybe what I tried to achieve here is to be more explicit. Right now "no arguments will be passed" is quite confusing especially since there's an `args` argument.

What about "no files names will be passed as arguments to the hook"?

Reply:
I would just replace the `arguments` -> `filenames` in the existing messaging

Reply:
Sure, done.